test/utils.spec.js

```diff
@@ -32,4 +32,171 @@ test('test the camelizeFiles function', () => {
     expect(obj).toHaveProperty(key);
     expect(obj[key]).toBe(value[1]);
   }
-});
\ No newline at end of file
+});
+
+const structures = [
```

Comment:
Can you add comments to name these different structures. Or alternatively, use an object with keys corresponding to the structure name so that if the tests fail, you can name the structure on which the test failed.